blog/2024-07-14-swarm-0.6-release.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Swarm 0.6 release"
+author: "Brent Yorgey"
+---
+
+The [Swarm](https://github.com/swarm-game/swarm/) development team is
+pleased to announce the latest (alpha) release of the game.
+
+It's been quite a while since our last release, so this one includes a
+lot.  Some of the biggest highlights include native Windows support,
+customizable keybindings, type synonyms and equirecursive types, a ton
+of small UI improvements, and a prototype tournament server where
+players can upload their solutions to challenge scenarios.
+
+We also have a [new Discord server][discord]---come say hi!
+
+Read on for a more in-depth discussion of some of the new features, or
+see the [CHANGELOG](https://github.com/swarm-game/swarm/blob/main/CHANGELOG.md) or even the [complete list of git
+commits](https://github.com/swarm-game/swarm/commits/main/?since=2023-11-01&until=2024-06-23).
+
+## What is it?
+
+As a reminder, Swarm is a 2D, open-world programming and resource
+gathering game with a strongly-typed, functional programming language
+and a unique upgrade system.
+
+## New challenge scenarios
+
+There are a number of new challenge scenarios included in this
+release, with screenshots of a few examples shown below.
+
+![](../gallery/beekeeping.png)
+
+![](../gallery/fishing.png)
+
+![](../gallery/dimsum.png)
+
+## Language improvements
+
+In terms of the Swarm programming language, this release has a few big
```

Comment:
I think it's worth to mention that the types in swarm language will now start with uppercase. But I leave that decision upto you.

Reply:
Ah, yes, good idea, I will add that.